CemEzgin/hw3/scripts/plus.js

```diff
@@ -0,0 +1,9 @@
+var Plus = function(firstNumber,lastNumber){
+	this.firstNumber = firstNumber;
+	this.lastNumber = lastNumber;
+}
+Plus.prototype.answer = function(){
+	var answer = parseFloat(this.firstNumber) + parseFloat(this.lastNumber);
+	document.getElementById('display').innerHTML="";
+	return buttonClick(answer);
```

Comment:
aslında answer metodu her class için farklı isimleri aynı sadece yeni class mı açmalıyım bunun için yoksa farklı isim mi gerekli ?